ros2pkg/ros2pkg/resource/ament_python/main.py.em

```diff
@@ -0,0 +1,7 @@
+import rclpy
+
+def main():
+    print("Hi from @project_name.")
+
+if __name__ == "__main__":
+    main()
```

Comment:
wrong return line
same below.

Reply:
Thanks for the feedback. All of the formatting issues in the template should be fixed now. I was able to create an example project and then run flake8 on each file and have the example pass.